### PR TITLE
FMWK-617 Update documentation

### PR DESCRIPTION
**Pull request by agrgr**

### CODE REVIEW

**Review by reugn — APPROVED** [review]

File: README.adoc [hunk 1]
```diff
@@ -111,7 +111,7 @@ Add the `spring-data-aerospike` Maven dependency:
 
 Notes:
 
-* It is recommended to use the latest version of Spring Data Aerospike, checkout Spring Data Aerospike
+* It is recommended to use the latest version, check out Spring Data Aerospike
```
Comment (rbotzer) [inline README.adoc:114]:
```suggestion
* It is recommended to use the latest version. Check out Spring Data Aerospike
```

File: README.adoc [hunk 2]
```diff
@@ -148,8 +148,8 @@ In order to configure Spring Data Aerospike you will need to create a configurat
 annotation.
 
 To set the connection details you can either override `getHosts()` and `nameSpace()` methods
-of the `AbstractAerospikeDataConfiguration` class or define `spring-data-aerospike.connection.hosts` and
-`spring-data-aerospike.connection.namespace` in `application.properties` file.
+of the `AbstractAerospikeDataConfiguration` class or define `spring.aerospike.hosts` and
+`spring.data.aerospike.namespace` in `application.properties` file.
```
Comment (rbotzer) [inline README.adoc:152]: I don't see documentation that talks about this change. Will that be discussed in the release notes and other markdown pages in the repo?

File: src/main/asciidoc/reference/aerospike-object-mapping.adoc [hunk 3]
```diff
@@ -29,49 +33,50 @@ The following table outlines the requirements for the `id` field:
...
 
 The following description outlines what type of conversion, if any, will be done on the property mapped to the `id` document field:
 
 * By default, the type of the field annotated with `@id` is turned into a `String` to be stored in Aerospike database.
 If the original type cannot be persisted (see xref:#configuration.keep-original-key-types[keepOriginalKeyTypes]
-for details), it must be convertible to `String` and will be stored in the database as such,
-then converted back to the original type when the object is read. This is transparent to the application
-but needs to be considered if using external tools like `AQL` to view the data.
+for details), it must be convertible to `String` and will be stored in the database as such, then converted back to the original type when the object is read.
+This is transparent to the application but needs to be considered if using external tools like `AQL` to view the data.
```
Comment (rbotzer) [inline src/main/asciidoc/reference/aerospike-object-mapping.adoc:46]:
```suggestion
This is transparent to the application but needs to be considered if using external tools like `AQL` and the Aerospike JDBC Driver to view the data.
```

**Review by rbotzer — APPROVED** [review]
I made a couple of small suggestions, but I'm pre-approving it.

File: README.adoc (hunk 2 above)
Comment (agrgr) [reply README.adoc:152]: @rbotzer The configuration is described in `configuration.adoc` file, it lists all properties

**Pull request merged**